sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/BatchAsyncClient.java

```diff
@@ -16009,12 +16693,8 @@ public Mono<Void> rebootNode(
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return A {@link Mono} that completes when a successful response is received.
      */
-    @Generated
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> rebootNode(String poolId, String nodeId) {
-        // Generated convenience method for rebootNodeWithResponse
-        RequestOptions requestOptions = new RequestOptions();
-        return rebootNodeWithResponse(poolId, nodeId, requestOptions).flatMap(FluxUtil::toMono);
+    public Mono<Void> rebootNode(String poolId, String nodeId, BatchBaseOptions options, NodeRebootOptions body) {
```

Comment:
Can we merge BaseBaseOptions and NodeRebootOptions into a single class? Maybe this means we should name NodeRebootOptions differently in the TypeSpec since the obvious name for such a unified class would be NodeRebootOptions.

Reply:
(This applies to all of the methods where we have arbitrary options in the request body rather than a resource)